app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <title>Findpropertyapp</title>
+
+    <link rel="stylesheet" href="static_pages.css.scss">
```

Comment:
No, this file should be added to 'application.css.scss', not here
It is probably already there if you have 'require_tree .'